style.css

```diff
@@ -0,0 +1,355 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+/* NAVBAR */
+.navbar {
+  width: 100%;
+  background-color: transparent;
+  display: flex;
+  flex-direction: row;
+  padding: 30px;
+  align-items: center;
+  z-index: 1;
+}
+
+.left-logo {
+  justify-self: start;
+  display: flex;
+  flex-direction: row;
+  padding-left: 230px;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+.left-logo a {
+  display: inline;
+  text-decoration: none;
+}
+
+.left-logo img {
+  width: 50px;
+}
+
+.left-logo p {
+  margin: 0;
+  font-weight: bolder;
+  color: white;
+}
+
+.right-menu {
+  justify-self: end;
+  padding-right: 230px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 15px;
+}
+
+.right-menu a {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 10px;
+  color: white;
+}
+
+.right-menu a:hover {
+  color: rgba(242, 148, 86, 255);
+}
+
+.right-menu i {
+  font-size: 5px;
+  color: rgba(242, 148, 86, 255);
+}
+
+/* WELCOME TEXT */
+.welcome-container {
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  margin-top: -120px;
+  color: white;
+  width: 100%;
+  height: 400px;
+}
+
+.welcome-container:before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: url("https://images.unsplash.com/photo-1525648199074-cee30ba79a4a?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=870&q=80");
```

Comment:
For the sake of consistency, I would download this image and store it into the project, the same way you did with the food images